### DISCUSSION TITLE
Prevent going back

### DISCUSSION THREAD

**Discussion by pke** [wix/react-native-navigation#7010] [Q&A] [answered]
If I wan to warn the user about changes made in the current screen is there a way to prevent back gestures, back button and Android back button?

**Comment by yogevbd** [comment, marked as answer]
Yes. see [handling the back button](https://wix.github.io/react-native-navigation/docs/stack#handling-the-back-button), [popStackOnPress](https://wix.github.io/react-native-navigation/api/options-backButton#popstackonpress) and [dismissModalOnPress](https://wix.github.io/react-native-navigation/api/options-hardwareBackButton#dismissmodalonpress).
